docs/dictionary/wordlist.txt

```diff
@@ -333,6 +333,8 @@ lang
 lastmodified
 leaveaspool
 lexically
+LifeScience
+LIFESCIENCE
```

Comment:
AFAIK, our spell check is not case sensitive so including a single `lifescience` would check for both `LifeScience` and `LIFESCIENCE`
```suggestion
lifescience
```

Reply:
Ok, let's try that!

Reply:
I've been using this command to fix the word list `sort docs/dictionary/wordlist.txt | tr A-Z a-z | sort -u -o docs/dictionary/wordlist.txt`.